### PR TITLE
AUI-1940 - make date validator accept brazilian format dd/mm/yyyy

### PR DESCRIPTION
**Pull request by linolaoj**
new pull request with test and source format

### CODE REVIEW

**Comment by jcampoy** [comment]
Hi @thiago-rocha , we need to fix this issue as soon as possible, there are a lot of users in spain reporting this (we have the same date format).
Could you review this PR ?
Best regards,

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-form-validator/tests/unit/js/tests.js [hunk 1]
```diff
@@ -190,6 +190,48 @@ YUI.add('aui-form-validator-tests', function(Y) {
             Y.Assert.isFalse(validator.hasErrors());
         },
 
+        /*
+         * Check if validator correctly validates dates in pt-br format dd/MM/yyyy
+         * @tests AUI-1940
+         */
+          'test date format': function() {
```
Comment (thiago-rocha) [inline src/aui-form-validator/tests/unit/js/tests.js]: @linolaoj  This test are falling on Mozilla Firefox. Could you please check this?

Comment (linolaoj) [inline src/aui-form-validator/tests/unit/js/tests.js]: Hi @thiago-rocha, the test case that failed is not regarding the date format  and is not caused by this commit so I will remove it and treat this case in another lps.

Comment (linolaoj) [inline src/aui-form-validator/tests/unit/js/tests.js]: Hi @thiago-rocha,  the issue was solved on the jsp code on LPS-56585 with https://github.com/liferay/liferay-portal-ee/commit/3da5ba4ae0f8d228a6693abb61a8e9bd80aaec6b

**Pull request closed without merging**